weave/Scripts/Score.cs

```diff
@@ -0,0 +1,72 @@
+using Godot;
+using GodotSharper.AutoGetNode;
+using System;
+using weave.Utils;
+
+namespace weave;
+
+public partial class Score : CanvasLayer
+{
+	private enum ScoringRule
```

Comment:
Indented with tabs, could you format it with CSharpier? (or just 4 spaces)

https://github.com/weave-game/weave#formatting